meteor-app/imports/ui/layouts/payment.js

```diff
@@ -0,0 +1,189 @@
+import React from 'react'
+import '../../lib/pinpayments.js'
```

Comment:
Yes this whole file "meteor-app/imports/ui/layouts/payment.js" is redundant as we are now using the button rather than our own payment form which is what this file was creating. 
Should I remove the file all together?

Reply:
If we are not going that way, then yes, delete it. It will stay in the git history anyway if we ever need to use it